systemd/google-cloud-ops-agent.service

```diff
@@ -18,6 +18,8 @@ Requires=google-cloud-ops-agent-fluent-bit.service google-cloud-ops-agent-opente
 
 [Service]
 Type=oneshot
+# Validate the config.
+ExecStartPre=@PREFIX@/libexec/google_cloud_ops_agent_engine -service=validate-only -in @SYSCONFDIR@/google-cloud-ops-agent/config.yaml
```

Comment:
With an empty string I believe the `-service` argument can go away entirely since empty string is the default value.

Reply:
Yes, my initial implementation did exactly that, but then we'd still need an explicit case for handling it in `GenerateFilesFromConfig`, and I thought it'd be confusing. But, given that you're thinking along the same lines, I can restore that.